examples/case-management.v1.test.js

```diff
@@ -0,0 +1,422 @@
+      originalLog(output);
+      done();
+    });
+    consoleWarnMock.mockImplementation(output => {
+      done(output);
+    });
+
+    expect(caseNumber).not.toBeNull();
+
+    // begin-removeWatchlist
+
+    const watchlistUsers = [
+      {realm: 'IBMid', user_id: 'abc@ibm.com'}
+    ]
+
+    const params = {
+      caseNumber: caseNumber,
+      watchlist: watchlistUsers,
+    };
+
+    caseManagementService.removeWatchlist(params)
+      .then(res => {
+        console.log(JSON.stringify(res.result, null, 2));
+      })
+      .catch(err => {
+        console.warn(err)
+      });
+
+    // end-removeWatchlist
+  });
+  test('addResource request example', done => {
+
+    consoleLogMock.mockImplementation(output => {
+      originalLog(output);
+      done();
+    });
+    consoleWarnMock.mockImplementation(output => {
+      done(output);
+    });
+
+    expect(caseNumber).not.toBeNull();
+    expect(resourceCrn).not.toBeNull();
+
+    // begin-addResource
+
+    const params = {
+      caseNumber: caseNumber,
+      crn: resourceCrn,
+      note: 'This resource is the service that is having the problem.',
+    };
+
+    caseManagementService.addResource(params)
+      .then(res => {
+        console.log(JSON.stringify(res.result, null, 2));
+      })
+      .catch(err => {
+        console.warn(err)
+      });
+
+    // end-addResource
+  });
+  test('uploadFile request example', done => {
+
+    consoleLogMock.mockImplementation(output => {
+      originalLog(output);
+      done();
+    });
+    consoleWarnMock.mockImplementation(output => {
+      done(output);
+    });
+
+    expect(caseNumber).not.toBeNull();
+
+    // begin-uploadFile
+
+    const exampleFileContent = 'This is the content of the file to upload.';
+
+    const fileWithMetadata = {
+      data: Buffer.from(exampleFileContent),
+      filename: 'example.log',
+      content_type: 'application/octet-stream',
+    };
+
+    const filesToUpload = [fileWithMetadata]
+
+    const params = {
+      caseNumber: caseNumber,
+      file: filesToUpload,
+    };
+
+    caseManagementService.uploadFile(params)
+      .then(res => {
+        attachmentId = res.result.id;
+        console.log(JSON.stringify(res.result, null, 2));
+      })
+      .catch(err => {
+        console.warn(err)
+      });
+
+    // end-uploadFile
+  });
+  test('downloadFile request example', done => {
+
+    consoleLogMock.mockImplementation(output => {
+      originalLog(output);
+      done();
+    });
+    consoleWarnMock.mockImplementation(output => {
+      done(output);
+    });
+
+    expect(caseNumber).not.toBeNull();
+    expect(attachmentId).not.toBeNull();
+
+    // begin-downloadFile
+
+    const params = {
+      caseNumber: caseNumber,
+      fileId: attachmentId,
+    };
+
+    caseManagementService.downloadFile(params)
+      .then(res => {
+
+        console.log(`Attachment content-type: ${res.headers['content-type']}\nAttachment contents: ${res.result.read()}`);
```

Comment:
I think it might be easier for a user to understand if we first initialize local variables `contentType` and `fileContents`, then just use them in the console.log() statement.  What do you think?
```suggestion
        const contentType = res.headers['content-type'];
        const fileContents = res.result.read();
        console.log(`Attachment content-type: ${contentType}\nAttachment contents: ${fileContents}`);
```